apps/test/unit/code-studio/components/progress/ResourcesDropdownTest.jsx

```diff
@@ -28,7 +28,7 @@ describe('TeacherResourcesDropdown', () => {
       wrapper.containsMatchingElement(
         <div>
           <DropdownButton
-            text={i18n.teacherResources()}
+            text={i18n.resources()}
```

Comment:
I'm guessing this is gonna want to stay the same, too (also one more below)

Reply:
🤦‍♀️ good catch! At least that one would've been caught be Drone though